crates/signs/src/bars.rs

```diff
@@ -19,12 +19,15 @@ use de_core::{
 };
 use de_objects::SolidObjects;
 
-use crate::{DISTANCE_FLAG_BIT, MAX_VISIBILITY_DISTANCE};
+use crate::{DISTANCE_FLAG_BIT, MAX_VISIBILITY_DISTANCE, UPDATE_TIMER_FLAG_BIT};
 
 /// Vertical distance in meters between the bar center and the top of the
 /// parent entity collider.
 const BAR_HEIGHT: f32 = 2.;
 
+/// Duration in seconds that a bar is visible when its value is updated.
+const UPDATE_VISIBILITY_DURATION: f32 = 3.;
```

Comment:
Nitpick: We can use `Duration` like this `Duration::from_secs(3)`. It is a const function and this way, it would be harder to misinterpret the units.